.babelrc

```diff
@@ -1,10 +1,14 @@
 {
+  "plugins": ["lodash"],
```

Comment:
I wonder if this still makes sense with the `_` needed to being available to external modules. 

Perhaps there can be some lazy logic to import `lodash` wholesale when external plugins are detected, before that only tree-shaken lodash functions could be included?

Reply:
Sounds like something worth investigating, but maybe for another PR, since this one's already been delayed a decent amount